friends/src/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import axios from 'axios';
+import Friend from './Friend';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    }
+  }
```

Comment:
Dustin briefly touched on it during lecture but you weren't part of our group yet when we discussed this - -it's absolutely fine the way it's written here, but because nothing is getting passed through the constructor/super here, you can actually declare state on the component without a constructor as `state={}` and still reference it as `this.state` later in the component. Just a way you can optimize your code and shine some light on what purpose the constructor and super actually do in React